### ISSUE TITLE
We want to write a OS

### ISSUE THREAD

**Issue by shaozikang** [Rbt-jiang/FoxOS#3]
Let us do this!

**Comment by Rbt-jiang** [comment]
yeah, Next term i will take a OS course and i also want to write a OS to examine my knowledge of OS, but i need time to learn the Computer Architect, it's will take me about 1 month.
